libs/cart/state/src/selectors/cart-item-entities/cart-item-entities.selectors.ts

```diff
@@ -21,6 +22,46 @@ export interface DaffCartItemEntitiesMemoizedSelectors<T extends DaffStatefulCar
 	selectIsCartItemOutOfStock: MemoizedSelectorWithProps<object, object, boolean>;
 	selectCartItemMutating: MemoizedSelector<object, boolean>;
 	selectCartItemState: MemoizedSelectorWithProps<object, object, DaffCartItemStateEnum>;
+  /**
+   * Selects the specified item's total discount for the entire row.
+   * Zero by default.
+   */
+	selectCartItemTotalDiscount: MemoizedSelectorWithProps<object, {id: DaffCartItem['item_id']}, number>;
```

Comment:
What is the difference between `selectCartItemTotalDiscount` and `selectCartItemDiscountedRowTotal`

Reply:
`selectCartItemDiscountedRowTotal` is a price, the row total minus the total discount

Reply:
maybe `selectCartItemRowTotalWithDiscount` is a clearer name?

Reply:
Another option is `selectCartItemTotalDiscountAmount` and `selectCartItemTotalDiscountedPrice`.

Reply:
oh, or just `selectCartItemTotalDiscount` and `selectCartItemTotalDiscountedPrice`

Reply:
bump